src/signer_node/node_parameters.rs

```diff
@@ -1,6 +1,7 @@
 use super::utils::sender_index;
 use crate::crypto::multi_party_schnorr::{Parameters, SharedKeys};
 use crate::crypto::vss::Vss;
+use crate::errors::Error;
```

Comment:
"cargo test" reports warning because Error is not used.

Reply:
fixed when rebasing.